tests/mtest/src/main/scala/tests/TestingWorkspaceSearch.scala

```diff
@@ -9,6 +9,7 @@ import scala.meta.inputs.Input
 import scala.meta.internal.metals.SemanticdbDefinition
 import scala.meta.internal.metals.WorkspaceSymbolInformation
 import scala.meta.internal.metals.WorkspaceSymbolQuery
+import scala.meta.internal.mtags.ScalametaCommonEnrichments.XtensionWorkspaceSymbolQuery
```

Comment:
Why is this needed?

Reply:
It provides extension methods which allows comparison against semanticdb SymbolInformation. I had to move it out of the `WorkspaceSymbol` class as it required semanticdb definitions which are not available in `mtags-shared`.